src/include/rados/librados.hpp

```diff
@@ -355,13 +356,21 @@ namespace librados
   class CEPH_RADOS_API ObjectWriteOperation : public ObjectOperation
   {
   protected:
-    time_t *pmtime;
+    ceph::real_time rt;
```

Comment:
changing members breaks the ABI - should add it to ObjectOperationImpl instead, move the implementation of mtime and mtime2 to librados.cc, and make sure the old version of mtime() is still ok to call